### PR TITLE
feat(users): added account verification and welcome mail template

### PR DESCRIPTION
**Pull request by Swiftwork**

### CODE REVIEW

File: internal/user/repo/postgres/postgres.go [hunk 1]
```diff
@@ -84,3 +108,57 @@ func (repo *Repository) Delete(id string) error {
...
+// CreateVerification creates a new verification code for a user
+func (repo *Repository) CreateVerification(id ksuid.KSUID) (string, error) {
+	return repo.CreateVerificationWithTransaction(id, nil)
+}
+
+// CreateVerificationWithTransaction creates a new verification code for a user
+func (repo *Repository) CreateVerificationWithTransaction(id ksuid.KSUID, transaction *sql.Tx) (string, error) {
+	verificationCode := fmt.Sprintf("%06d", rand.Intn(999999))
+	var err error
+	if transaction == nil {
```
Comment (lowekoz) [inline internal/user/repo/postgres/postgres.go:121]: branches executes same code, shorten?

Comment (Swiftwork) [reply internal/user/repo/postgres/postgres.go:121]: @lowekoz can you elaborate?

Comment (Swiftwork) [reply internal/user/repo/postgres/postgres.go:121]: ah woops, I think I realize now. Ill check ^^

Comment (Swiftwork) [reply internal/user/repo/postgres/postgres.go]: fixed

File: db/schema.sql [hunk 2]
```diff
@@ -32,6 +32,19 @@ CREATE TABLE user_verifications (
 		FOREIGN KEY(user_id)
 			REFERENCES users(id)
 				ON DELETE CASCADE
+
```
Comment (lowekoz) [inline db/schema.sql:35]: missing );

File: db/seed.sql [hunk 3]
```diff
@@ -30,6 +30,14 @@ INSERT INTO user_verifications(user_id, code) VALUES (
...
 
+INSERT INTO user_permissions (user_id) VALUES (
+  '1pJ1jjtx5mP3XThvQlQWzwrEjmA'
+), (
+  '1pJ1md0qvG9LJHuduH6LKe1BsA8'
+), (
+  '1pRCqPTClWxhfn08u8NfjefOgLz'
+);
+
 INSERT INTO pantries (id, name, invite_code) VALUES (
```
Comment (lowekoz) [inline db/seed.sql:41]: missing field invite_code in pantries

**Review by lowekoz — CHANGES_REQUESTED** [review]
database files seems to be in disarray (more than what I have commented on). Try and run "db" command to see remaining conflicts. Also, I am unable to open this branch's .yaml file in stoplight, is this true for you as well?

**Pull request merged**